docs/core-concepts/versioning.md

```diff
@@ -1,6 +1,154 @@
-# 🔧 .NET Core Versioning
+.NET Core Versioning
+==================== 
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+.NET Core is a [platform of NuGet packages](packages.md), a set of frameworks and distributed as a unit. Each of these platform layers can be versioned separately for product agility and to accurately describe product changes. While there is significant versioning flexibility, there is a desire to version the platform as a unit to make the product easier to understand.
 
+The product is somewhat unique, being described and delivered via a package manager (NuGet) as packages. While you typically acquire .NET Core as a standalone SDK, the SDK is largely a convenience experience over NuGet packages and therefore not distinct from packages. As a result, versioning is first and foremost in terms of packages and other versioning experiences follow from there.
+
+Semantic Versioning
+===================
+
+.NET Core uses [Semantic Versioning (SemVer)](http://semver.org/), adopting the use of major.minor.patch versioning, using the various parts of the version number to describe the degree and kind of change.
+
+The following versioning template is generally applied to .NET Core. There are cases where it has been adapted to fit with existing versioning. These cases are described later in this document. For example, frameworks only have a two part version number and are only intended to represent platform and API capabilities, which aligns with major/minor versioning.
+
+Versioning Form
+---------------
+
+MAJOR.MINOR.PATCH[-PRERELEASE.BUILDNUMBER]
+
+Decision Tree
+-------------
+
+MAJOR when:
+  - drop support for a platform
+  - adopt a newer MAJOR version of an existing dependency 
+  - disable a compatibility quirk off by default
+
+MINOR when:
+  - add public API surface area 
+  - add new behavior
+  - adopt a newer MINOR version of an existing dependency
+  - introduce a new dependency 
+  
+PATCH when:
+  - make bug fixes
+  - add support for a newer platform
+  - adopt a newer PATCH version of an existing dependency
+  - any other change
```

Comment:
This makes it sound like the default field to update is PATCH, which I don't think is what you mean. I'll suggest "any other change that doesn't trigger MINOR or MAJOR version updates". It's not perfect, but might spark an idea.